### PR TITLE
Weather app "working"

### PR DESCRIPTION
**Pull request by JasonStieber**
Working see its working at the bat-trail

### CODE REVIEW

**Comment by flyylikeaduck** [comment]
Component lifecycle bug -- this.state.lng & lat are being passed into component correctly, but not updating the Forecast component. 

(WIP)

File: client/src/components/Weather.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        lat: nextProps.mapCenter.lat,
+        lng: nextProps.mapCenter.lng,
+        trailName: nextProps.trailName
+      })
+    }
+  }
+
+  render(){
+    return (
+      <Forecast latitude={37} longitude={-122} name={'the Bat-Trail'} />
```
Comment (clarissachin) [inline client/src/components/Weather.jsx:26]: Group acknowledgement that lat and long are hardcoded for component to work correctly for now.

File: client/src/page-components/Trail.jsx [hunk 2]
```diff
@@ -55,8 +51,11 @@ class Trail extends React.Component {
     });
   }
 
+  componentDidUpdate() {
```
Comment (clarissachin) [inline client/src/page-components/Trail.jsx:54]: Group acknowledgment this is blank for now and intention was code is to be deleted.

File: database/config.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
     "development": {
-        "username": "student",
-        "password": null,
+        "username": "root",
```
Comment (clarissachin) [inline database/config.json:3]: Group note: update development object to have "student" as username and password as "student" acc: to original master repo.

File: database/config.json [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+        "password": "",
         "database": "trektracker",
         "host": "localhost",
         "dialect": "mysql"
     },
     "test": {
         "username": "root",
-        "password": null,
+        "password": "",
```
Comment (clarissachin) [inline database/config.json:11]: Group note: acknowledgement we will update test object to have "root" as username and password as null acc: to original master repo.

**Pull request merged**